### DISCUSSION TITLE
Welcome to frogo-notification Discussions!

### DISCUSSION THREAD

**Discussion by amirisback** [amirisback/frogo-notification#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by supersu-man** [comment]
Possible to add a button to the notification ? That's the only thing I am missing. Thnaks for this amazing library.

**Reply by amirisback** [reply]
thank you very much, for now it can't, maybe I'll put it in the issue
